client/state/ui/test/selectors.js

```diff
@@ -230,4 +231,38 @@ describe( 'selectors', () => {
 			expect( siteSection ).toBe( true );
 		} );
 	} );
+
+	describe( 'isBlazeEnabled()', () => {
+		test( 'should return false if no siteId is defined', () => {
+			const siteId = null;
+			const state = {
+				sites: {
+					items: {
+						[ siteId ]: {},
+					},
```

Comment:
This is an odd, impossible scenario - you'll never see a `null` siteID in the state. I'd suggest changing this to just an empty object:

```suggestion
					items: {},
```